src/git-town-append

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+source "$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/helpers/helpers.sh"
+
+
+function ensure_has_target_branch {
+  if [ -z "$target_branch_name" ]; then
+    echo_error_header
+    echo_error "No branch name provided."
+    exit_with_error newline
+  fi
+}
+
+
+function preconditions {
+  target_branch_name=$1
+  ensure_has_target_branch
+
+  parent_branch_name=$INITIAL_BRANCH_NAME
+  if [ "$parent_branch_name" != "$MAIN_BRANCH_NAME" ]; then
+    ensure_knows_parent_branches "$parent_branch_name"
+  fi
+
+  if [ "$(has_remote_url)" = true ]; then
+    fetch
+  fi
+
+  ensure_does_not_have_branch "$target_branch_name"
+
+  export RUN_IN_GIT_ROOT=true
+  export STASH_OPEN_CHANGES=true
+}
+
+
+function steps {
+  echo "create_and_checkout_feature_branch $target_branch_name $parent_branch_name"
```

Comment:
We needed to sync in `git hack`, because it cuts off the master branch and that should be up to date. Here the developer continues ongoing development on a child branch. I see less need to sync here. The developer can run `git sync` at any time if they want before this. What do you think about that?